spec/chromium-spec.js

```diff
@@ -735,27 +735,6 @@ describe('chromium feature', () => {
   })
 
   describe('window.postMessage', () => {
-    it('sets the source and origin correctly', (done) => {
```

Comment:
Can you pull out this change into a separate PR, would help fixing flakes on other PR's that are against master ex: https://github.com/electron/electron/pull/18726/checks?check_run_id=145836138. Thanks!